ingestion-beam/src/main/java/com/mozilla/telemetry/contextualservices/VerifyMetadata.java

```diff
@@ -69,7 +69,7 @@ public Result<PCollection<PubsubMessage>, PubsubMessage> expand(
                 String.format("Invalid Firefox version: %s", version), "user_agent_version");
           }
 
-          // Verify release channel
+          // Verify release channel for specific doctype
```

Comment:
All Fenix pings are getting rejected in VerifyMetadata with "Invalid user agent: null" so we'll need more refactoring here to make sure the desktop-specific checks are only applied if document namespace is "contextual-services".